testsuite/tests/singlecluster/conftest.py

```diff
@@ -41,7 +41,7 @@ def authorization_name(blame):
 @pytest.fixture(scope="module")
 def authorization(request, kuadrant, route, gateway, blame, cluster, label):  # pylint: disable=unused-argument
     """Authorization object (In case of Kuadrant AuthPolicy)"""
-    target_ref = request.getfixturevalue(getattr(request, "param", "route"))
+    target_ref = request.getfixturevalue(getattr(request, "param", "gateway"))
```

Comment:
This will indeed make all tests in `.../gateway/` module have polices that attach to Gateway but so will all tests in other modules which do not expect this :sweat_smile: 
Only tests in `.../gateway/` module should have the Gateway attachment the default. I would do changes in the `.../gateway/conftest.py` to achieve this